config/manager/kustomization.yaml

```diff
@@ -3,7 +3,7 @@ resources:
 
 images:
 - name: compliance-operator
-  newName: quay.io/compliance-operator/compliance-operator
+  newName: image-registry.openshift-image-registry.svc:5000/openshift/compliance-operator
```

Comment:
@Vincent056 just a FYI, this substitution is local cause of the make targets, and shouldn't be committed. If you run `make clean` it'll restore the default.